src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -281,8 +281,27 @@ private async Task<FunctionMetadataResponse> GetFunctionMetadataAsync(string fun
 
                 foreach (var func in functionMetadataList)
                 {
-                    response.FunctionMetadataResults.Add(func);
+                    if (func.RawBindings is null)
```

Comment:
Do you need to check this contains at least 1 item? `if (func.RawBindings?.Any() != true)`